src/views/product-downloads-view/components/featured-learn-cards-section/index.tsx

```diff
@@ -44,7 +44,7 @@ const FeaturedLearnCardsSection = ({
 						(collectionCardProps: FeaturedCollectionCard) => {
 							return (
 								<li key={collectionCardProps.id}>
-									<CollectionCard {...collectionCardProps} />
+									<CollectionCardWithAuthElements {...collectionCardProps} />
```

Comment:
In all places where we previously used `CollectionCard`, we now use `CollectionCardWithAuth`. This is one of those places, & there are a few more in other files.

> **Note**: I think there are very few (if any?) scenarios where we'll want to use the collection card without auth 🤔 So, an alternative here could be to change the default export of `components/collection-card` to `CollectionCardWithAuth`, and maybe rename things so the "with auth" version is called `CollectionCard` and the "without auth, presentational" version is called `CollectionCardPresentational` or something.
> 
> Open to ideas, figured for this PR might as well stick to convention established with `TutorialCard` 👍